sdk/communication/communication-common/src/identifierModels.ts

```diff
@@ -262,6 +344,41 @@ export const createIdentifierFromRawId = (rawId: string): CommunicationIdentifie
     case "8:dod-acs:":
     case "8:gcch-acs:":
       return { kind: "communicationUser", communicationUserId: rawId };
+    case "28:gcch-global:":
+      return {
+        kind: "microsoftTeamsBot",
+        microsoftTeamsBotId: suffix,
+        isGlobal: true,
+        cloud: "gcch",
+      };
+    case "28:orgid:":
+      return {
+        kind: "microsoftTeamsBot",
+        microsoftTeamsBotId: suffix,
+        isGlobal: false,
+        cloud: "public",
+      };  
+    case "28:dod-global:":
+      return {
+        kind: "microsoftTeamsBot",
+        microsoftTeamsBotId: suffix,
+        isGlobal: true,
+        cloud: "dod",
+      };
+    case "28:gcch:":
+      return {
+        kind: "microsoftTeamsBot",
+        microsoftTeamsBotId: suffix,
+        isGlobal: false,
+        cloud: "gcch",
+      };     
+    case "28:dod:":
+      return {
+        kind: "microsoftTeamsBot",
+        microsoftTeamsBotId: suffix,
+        isGlobal: false,
+        cloud: "dod",
+      };  
```

Comment:
Since we are using same attributes in all the cases, would make sense to create a method that returns the object and then call that method in each cases passing required values.
Something like this :
```
const buildIdentifier(kind:string, id:string, cloud:string, flag:boolean) {
  return {
          kind: kind,
          microsoftTeamsBotId: id,
          isGlobal: flag,
          cloud: cloud,
        };
  }
```

```
case "28:dod:":
      return buildIdentifier("microsoftTeamsBot", suffix, "dod", false)
```
     
We can do the same for MicrosoftTeamsUser as well.

Reply:
I would suggest pass only `cloud` and `flag` as args, as the rest of the attrs are same

Reply:
Done!